[BUG] Failing test for release 8.1.3: `AUTH errored inside MULTI`

**Describe the bug**

```
!!! WARNING The following tests failed:

*** [err]: AUTH errored inside MULTI will add the reply in tests/unit/multi.tcl
Expected an error matching 'WRONGPASS invalid username-password pair or user is disabled.' but got '' (context: type eval line 5 cmd {assert_error {WRONGPASS invalid username-password pair or user is disabled.} {r exec}} proc ::test)
Cleanup: may take some time... OK
make[1]: *** [Makefile:580: test] Error 1
make[1]: Leaving directory '/build/valkey/src/valkey-8.1.3/src'
make: *** [Makefile:6: test] Error 2
make: Leaving directory '/build/valkey/src/valkey-8.1.3'
```

8.1.2 tests are good.

**To reproduce**

Build ArchLinux PKGBUILD with `pkgctl build` (and bumped version).

**Expected behavior**

Tests pass.

**Additional information**

-

--- a/tests/unit/multi.tcl
+++ b/tests/unit/multi.tcl
@@ -900,6 +900,7 @@ start_server {tags {"multi"}} {
         r watch b{t} a{t}
         r flushall
         r ping
+        r unwatch
      }
 
     test "AUTH errored inside MULTI will add the reply" {

